document/CoreFormat.mdk

```diff
@@ -1409,6 +1425,165 @@ A `annotatedCodeLocation` object may contain a property named `tags`, whose valu
 An `annotatedCodeLocation` object may contain a property named `properties`, whose value is a property bag.
 This allows tools to include information about the entry that is not explicitly specified in the SARIF format.
 
+## `rule` object { #rule }
+
+### General { #rule-General }
+
+A `rule` object contains information that describes a rule.
+
+### Constraints
+
+Either the `shortDescription` property (&sect;[#rule-shortDescription])
+or the `fullDescription` property (&sect;[#rule-fullDescription])
+or both must be present.
+
+### `id` property { #rule-id }
+
+A `rule` object shall contain a property named `id` whose value is a string containing
+a stable, opaque identifier for the rule.
+
+~ IsoExample
+`"CA2101"`
```

Comment:
Because the value of the `id` property, of which this is an example, is a (quoted) string.